remoter-server/README.md

```diff
@@ -14,4 +14,4 @@ with the argument being the JSON-ified string of the task that is to be executed
 
```

Comment:
also, it will be very useful to specify in this README that how the task should be formatted is written in the schema in the API and put a link to the schemas for the pipeline tasks

Reply:
Done.